src/palindrom.js

```diff
@@ -331,8 +331,13 @@ const Palindrom = (() => {
         onSocketOpenCallback && onSocketOpenCallback(event);
       };
       this._ws.onmessage = event => {
-        const parsedMessage = JSON.parse(event.data);
-        this.onReceive(parsedMessage, this._ws.url, 'WS');
+        try {
+          const parsedMessage = JSON.parse(event.data);
+          this.onReceive(parsedMessage, this._ws.url, 'WS');
+        }
+        catch (e) {
+          this.onFatalError(event.data, upgradeURL, 'WS');
+        }
```

Comment:
Can you add a screenshot of how does it look with the server-side change from https://github.com/Starcounter/Home/issues/340?

@alemoi sends the stack trace in the text message and then sends the closing frame with the reason. Does it look decently readable in Palindrom or is it displayed as two verbose errors?

Reply:
Yes, 2 frames, one message string with error and second Close frame that says: "The error stack trace is sent in a previous message."

Reply:
I couldn't see the nice error message due to a `palindrom-polymer-client` limitation. 

Here is how it looks:
![image](https://user-images.githubusercontent.com/17054134/34102761-8b65a232-e3ea-11e7-9709-eb763133d951.png)

And that's because `palindrom-polymer-client` constructs the error message on its own and neglects `data` property from the event. 

See:
![image](https://user-images.githubusercontent.com/17054134/34102945-356ffc78-e3eb-11e7-80ef-86089d49b879.png)